### DISCUSSION TITLE
Is AH2 compatible with the new v21.0a1-Omega

### DISCUSSION THREAD

**Discussion by xyzfre** [jurialmunkey/skin.arctic.horizon.2#917] [Discussion]
I wanted to try the new update v21 Omega but wanted to be safe if it has any conflict or issues with AH2

**Comment by Nuklear92** [comment]
I can confirm that it works pretty good with Omega, I've tried it before and never had any issues with it. I personally installed the stable 20.1 again, until we get into the RC stage of Omega to go all out on it and say good bye to Nexus.

**Reply by xyzfre** [reply]
Thank you for your response really appreciated maybe I'll play around with v21 Omega for a bit

**Comment by jurialmunkey** [comment]
As far as I'm aware, there aren't any major skinning engine changes yet so it should work fine.

**Reply by tyshiro** [reply]
Favorites seems to be off-line

**Reply by qazhw** [reply]
Can't update normally

**Comment by xyzfre** [comment]
Thank you @jurialmunkey for your response and for your AMAZING work AH2 just gets better 💯💯
